README.md

```diff
@@ -10,25 +10,29 @@ Interactive navigable audio visualization using Web Audio and Canvas.
 
 See a [tutorial](https://wavesurfer-js.org/docs) and [examples](https://wavesurfer-js.org/examples) on [wavesurfer-js.org](https://wavesurfer-js.org).
 
-## Browser support
-wavesurfer.js works only in [modern browsers supporting Web Audio](http://caniuse.com/audio-api).
+## Questions
+Have a question about integrating wavesurfer.js on your website? Feel free to ask in our forum: https://github.com/wavesurfer-js/wavesurfer.js/discussions/categories/q-a
 
-It will fallback to Audio Element without graphics in other browsers (IE 11 and lower).
+## Quick start
 
-## FAQ
-### Can the audio start playing before the waveform is drawn?
-Yes, if you use the `backend: 'MediaElement'` option. See here: https://wavesurfer-js.org/example/audio-element/. The audio will start playing as you press play. A thin line will be displayed until the whole audio file is downloaded and decoded to draw the waveform.
+```
+npm install wavesurfer.js --save
+
+# or
+
+yarn add wavesurfer.js
 
-### Can drawing be done as file loads?
-No. Web Audio needs the whole file to decode it in the browser. You can however load pre-decoded waveform data to draw the waveform immediately. See here: https://wavesurfer-js.org/example/audio-element/ (the "Pre-recoded Peaks" section).
+# or
 
-## API in examples
+<script src="https://cdnjs.cloudflare.com/ajax/libs/wavesurfer.js/6.4.0/wavesurfer.min.js" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```

Comment:
- This means we need to update the version number everytime a new release is made
- We always used unpkg.com instead of cloudflare (which allows you to use the latest version instead of specifying a version nr)

Reply:
Sounds good, will change this.